srcpkgs/xplr/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'xplr'
+pkgname=xplr
+version=0.20.0
+revision=1
+build_style=cargo
+short_desc="Hackable, minimal, fast TUI file explorer"
+maintainer="Muhammad Nauman Raza <thedevraza@proton.me>"
+license="MIT"
+homepage="https://github.com/sayanarjit/xplr"
+distfiles="https://github.com/sayanarijit/xplr/archive/refs/tags/v${version}.tar.gz"
+checksum=c4d63d9e1e313eeeb2e6d8d17e30b18ee4b8be01c419f08a89959fe5a4a09ac0
+
+do_install() {
+	     vlicense LICENSE
+	     vdoc README.md
+	     vmkdir usr/share/doc/xplr/
```

Comment:
this dir should already have been created by vdoc

Reply:
Removed creation of the ;usr/share/doc/xplr' directory.